examples/query_transaction/query_tx.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	broadcast_client "github.com/bitcoin-sv/go-broadcast-client/broadcast/broadcast-client"
+)
+
+func main() {
+	token := ""
+	apiURL := "https://tapi.taal.com/arc"
+	hex := "9c5f5244ee45e8c3213521c1d1d5df265d6c74fb108961a876917073d65fef14"
```

Comment:
Just a question:
Isn't that pointing to your private transaction or some transaction that could contain some private staff about you?

Reply:
No, it's a random transaction